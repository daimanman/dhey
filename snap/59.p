{"C":1,"Duration":65854324,"EndTime":1535337562683242100,"Err":"","FileId":"2","InputFileName":"file","Method":"POST","N":1,"P":{"info":"TEssssssssskkkkkk颠三倒四ST","name":"sssssDXwwM"},"Payload":"","QH":{},"Remark":"压力测试","ReqNums":1,"StartTime":1535337562617387800,"Status":1,"TaskId":59,"Type":"FILE","Url":"http://192.168.227.129:9090/api/v1/oss","Z":""}