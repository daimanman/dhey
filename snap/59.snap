{"AvgTotal":0.0199465,"Fastest":0.0199465,"Slowest":0.0199465,"Average":0.0199465,"Rps":50.134108740881864,"AvgConn":0.0010123,"AvgDNS":0,"AvgReq":0.0009504,"AvgRes":0,"AvgDelay":0.0179838,"ConnMax":0,"ConnMin":0,"DnsMax":0,"DnsMin":0,"ReqMin":0.0009504,"ReqMax":0.0009504,"ResMin":0.0009504,"ResMax":0,"DelayMax":0.0179838,"DelayMin":0.0179838,"Lats":[0.0199465],"ConnLats":[0],"DnsLats":[0],"ReqLats":[0.0009504],"ResLats":[0],"DelayLats":[0.0179838],"Total":19946500,"ErrorDist":{},"StatusCodeDist":{"200":1},"SizeTotal":12,"SizeReq":12,"NumRes":1,"LatencyDistribution":[{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0}],"Histogram":[{"Mark":0.0199465,"Count":1,"Frequentcy":1},{"Mark":0.0199465,"Count":0,"Frequentcy":0},{"Mark":0.0199465,"Count":0,"Frequentcy":0},{"Mark":0.0199465,"Count":0,"Frequentcy":0},{"Mark":0.0199465,"Count":0,"Frequentcy":0},{"Mark":0.0199465,"Count":0,"Frequentcy":0},{"Mark":0.0199465,"Count":0,"Frequentcy":0},{"Mark":0.0199465,"Count":0,"Frequentcy":0},{"Mark":0.0199465,"Count":0,"Frequentcy":0},{"Mark":0.0199465,"Count":0,"Frequentcy":0},{"Mark":0.0199465,"Count":0,"Frequentcy":0}],"TaskId":59,"Duration":0.0199465}